NetCoreDemos/wwwroot/css/DemosStyles/TreeList/FormEditing.css

```diff
@@ -1,3 +1,4 @@
-﻿#tree-list-demo {
-    min-height: 700px;
-}
\ No newline at end of file
+﻿#employees {
+    max-height: 640px;
+}
+  
```

Comment:
Let's fix the ```end of file``` here

Reply:
I don't know why the empty 4th line isn't okay here O_o
But still, it's linter issue